pkgs/top-level/linux-kernels.nix

```
@@ -380,6 +380,8 @@ in {
 
     lenovo-legion-module = callPackage ../os-specific/linux/lenovo-legion { };
 
+    linux-gpib-kernel = callPackage ../applications/science/electronics/linux-gpib/linux-gpib-kernel { };
```

```suggestion
    linux-gpib = callPackage ../applications/science/electronics/linux-gpib/linux-gpib { };
```
we could also name the packages the same since they are in different package sets

---

Updating the package name but retaining package directory name.